npm/README.md

```diff
@@ -12,56 +12,10 @@ Install with [npm](https://www.npmjs.com/) as a global module:
 $ npm install --global @openfaas/faas-cli
```

Comment:
On my computer, it tired to go into `/usr/local/lib/node_modules` (or a similar path and was access denied, even if run as sudo) 

Where does `~/.npm-global/bin/` come from?

Reply:
I realized as soon as I read your comment, that I have a `prefix` set in my `.npmrc` file. That's where that path comes from. I'll probably just remove that line mentioning the location since it's the same as other global npm installs.
Odd you had a permissions issue. I'll look into that as well.

Reply:
I'll await that change.